tests/bluetooth/tester/src/bttester.h

```diff
@@ -708,6 +708,16 @@ struct mesh_input_string_cmd {
 	u8_t string[0];
 } __packed;
 
+#define MESH_IVU_TEST_MODE		0x08
+struct mesh_ivu_test_mode_cmd {
+	u8_t toggle;
+} __packed;
+
+#define MESH_IVU_TOGGLE_STATE		0x09
+struct mesh_ivu_toggle_state_cmd {
+	u8_t toggle;
```

Comment:
I think this command shouldn't take any parameter,  toggle by definition is changing bit value and tester doesn't really has means to to control it.

Just update BTP spec if needed with those.

Reply:
done